tests/test_gas_costs.py

```diff
@@ -85,7 +85,7 @@ def test_cost_transfer_2_mediators(
     A, B, C, D, *rest = accounts
     tx_hash = contract.functions.transfer(D, 50, 6, [B, C, D]).transact({"from": A})
     gas_cost = get_gas_costs(web3, tx_hash)
-    report_gas_costs(table, "2 hop transfer", gas_cost, limit=78000)
+    report_gas_costs(table, "2 hop transfer", gas_cost, limit=80000)
```

Comment:
Was this necessary because to the compiler upgrade?
Seems wrong that it gets more expensive with a newer version.

Reply:
Yes, it is only due to the compiler upgrade. It also seemed wrong to me but as it is only 2k gas (actually 1.1k gas if I remember correctly) I did not worry too much.